src/Microsoft.CSharp/src/Configurations.props

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <BuildConfigurations>
+      netstandard1.7|AnyOS;
```

Comment:
Should we change most of these to `netcoreapp|AnyOS` or however we are going to encode that here? Only a select few projects are going to actually have `netstandard` configurations in the new world.